### PR TITLE
Add client docs for using fully managed sdxl endpoint

### PR DESCRIPTION
**Pull request by bddppq**

### CODE REVIEW

File: advanced/sdxl/README.md [hunk 1]
```diff
@@ -8,6 +8,72 @@ There are two ways to access SDXL model:
...
+seed = 1234
+
+
+# Directly using urls to pass images
+image_bytes = c.inpaint(image=img_url, mask_image=mask_url, prompt=prompt, seed=seed)
+with open("inpaint_url.png", "wb") as f:
+    f.write(image_bytes)
+
+# Or use FileParam to send image files:
+img_content = requests.get(img_url).content
```
Comment (Yangqing) [inline advanced/sdxl/README.md:53]: nit: need `import requests`

File: advanced/sdxl/README.md [hunk 2]
```diff
@@ -8,6 +8,72 @@ There are two ways to access SDXL model:
 
 Lepton provides the SDXL model as a fully managed api endpoints at https://sdxl.lepton.run. Users can easily use the lepton Python client or existing https request tool to generate high resolution realistic images right away.
 
+Creating the client:
+```python
+from leptonai.client import Client
```
Comment (Yangqing) [inline advanced/sdxl/README.md:13]: nit: from leptonai.photon import FileParam

File: advanced/sdxl/README.md (hunk 1 above)
Comment (Yangqing) [reply advanced/sdxl/README.md:53]: More nit: `import base64`

Comment (bobmayuze) [reply advanced/sdxl/README.md:53]:
```
import requests
from leptonai.photon import FileParam
import base64
```

**Pull request merged**